frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/AddPolicyModal.tsx

```diff
@@ -70,7 +103,20 @@ const AddPolicyModal = ({
     teamId,
   ]);
 
-  const policiesAvailable = DEFAULT_POLICIES.map((policy: IPolicyNew) => {
+  const onPlatformFilterChange = (platformSelected: SelectedPlatform) => {
+    if (platformSelected === "all") {
+      setFilteredPolicies(DEFAULT_POLICIES);
+    } else {
+      // Note: Default policies currently map to a single platform
+      const policiesFilteredByPlatform = DEFAULT_POLICIES.filter((policy) => {
+        return policy.platform === platformSelected;
+      });
+      setFilteredPolicies(policiesFilteredByPlatform);
+    }
+    setPlatform(platformSelected);
+  };
+
+  const renderFilteredPolicies = filteredPolicies.map((policy: IPolicyNew) => {
```

Comment:
Thanks @RachelElysia . I suggested `rendered`... since this is an array of rendered policies. `render`... implies this is a function that one would call, which it's not.